client/jetpack-onboarding/steps/site-title.jsx

```diff
@@ -92,7 +96,7 @@ class JetpackOnboardingSiteTitleStep extends React.PureComponent {
 					</form>
 				</Card>
 
-				<JetpackOnboardingDisclaimer />
+				<JetpackOnboardingDisclaimer handleTosClick={ this.handleTosClick } />
```

Comment:
I think I'd rather pass the 'plain' `recordJpoEvent` action (rather than the event-name bound `handleTosClick`). My rationale is that the event name is specific to the Disclaimer component and should hence be encapsulated in it.

Reply:
Funny, that was my initial approach, but decided to go for simplicity (because your approach required changing the disclaimer to a `Component`).

Addressed by https://github.com/Automattic/wp-calypso/pull/22382/commits/1ee67ee036025cda8df62526f8d7a45270e13416

Reply:
Yeah, I thought that might be the underlying motivation. Anyway, thanks for addressing, feels cleaner now! 👍